src/components/ErrorPage.jsx

```diff
@@ -0,0 +1,3 @@
+export default function ErrorPage({ error }) {
+  return <p>{error.message}</p>;
```

Comment:
At the moment this error message is showing as undefined, so may need to take a look at this if you want to keep it this way